tests/internal/e2e/test/wdio.conf.ts

```diff
@@ -750,3 +866,6 @@ ENVIRO.END
   // onReload: function(oldSessionId, newSessionId) {
   // }
 };
+function copyDirectory(sourceFile: string, destinationFile: string) {
```

Comment:
Should we remove this if it's neither used nor implemented

Reply:
Removed in 7c690c6fda33b48239ce5fb7aeb15024a3df53d6.